orm.go

```diff
@@ -438,10 +435,17 @@ func (fo *FieldOptions) addOptions(options ...interface{}) error {
 // FieldOption is used to pass an option to index.Field function.
 type FieldOption func(options *FieldOptions) error
 
-// OptFieldCacheSize sets the cache size.
-func OptFieldCacheSize(size uint) FieldOption {
+// OptFieldInt adds a set field.
+// Specify CacheTypeDefault for the default cache type.
+// Specify 0 for the default cache size.
```

Comment:
It seems weird to use the default cache size when the value provided is 0. At that point, why not provide a `CacheSizeDefault` const for consistency with the `CacheTypeDefault ` option? I think if someone sets `cacheSize` to 0 it should effectively act like `CacheTypeNone`.

Reply:
Added `CacheSizeDefault` const.